### DISCUSSION TITLE
Feature Request: Terminate Server Task via Plugin

### DISCUSSION THREAD

**Discussion by Pete3030** [apohl79/audiogridder#810] [Help]
Hi, I often have the problem that restarting the Server does not work via the Plugin when the Server App has already crashed. I then have to go directly on the server computer and terminate the Audiogridder Server Task via Task Manager (PC) or Activity Monitor (Mac). Then it automatically restarts again..
It would be much more convenient if I could to this directly via the Plugin (Terminate Audiogridder task). Thanks for your hard work!

Regards,
Peter

**Comment by CaBuffalo** [comment]
Hi, Peter:

If the server crashes, the plugin won't be able to talk to the server.
Restarting the server host is the only way to do it.

On Sun, Nov 21, 2021 at 11:38 AM Pete3030 ***@***.***> wrote:

> Hi, I often have the problem that restarting the Server does not work via
> the Plugin when the Server App has already crashed. I then have to go
> directly on the server computer and terminate the Audiogridder Server Task
> via Task Manager (PC) or Activity Monitor (Mac). Then it automatically
> restarts again..
> It would be much more convenient if I could to this directly via the
> Plugin (Terminate Audiogridder task). Thanks for your hard work!
>
> Regards,
> Peter
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/810>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6CIOMM33XYWQEESK3DUNFDBPANCNFSM5IPPZTGQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by Pete3030** [reply]
Ah ok, my hope was that with some kind of programming wizardry it would be possible to terminate the task of the server computer via the plugin..;)
